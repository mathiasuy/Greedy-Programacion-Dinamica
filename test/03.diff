1,2c1,21
< 1>Cantidad: 9
< 2>Fin.
---
> ==4608== Use of uninitialised value of size 8
> ==4608==    at 0x109D77: max_cantidad(intervalo_t const*, unsigned int) (in /mnt/c/p3/tareap3ant/principal)
> ==4608==    by 0x1094D8: main (in /mnt/c/p3/tareap3ant/principal)
> ==4608== 
> ==4608== Invalid write of size 8
> ==4608==    at 0x109D77: max_cantidad(intervalo_t const*, unsigned int) (in /mnt/c/p3/tareap3ant/principal)
> ==4608==    by 0x1094D8: main (in /mnt/c/p3/tareap3ant/principal)
> ==4608==  Address 0x0 is not stack'd, malloc'd or (recently) free'd
> ==4608== 
> ==4608== 
> ==4608== Process terminating with default action of signal 11 (SIGSEGV)
> ==4608==  Access not within mapped region at address 0x0
> ==4608==    at 0x109D77: max_cantidad(intervalo_t const*, unsigned int) (in /mnt/c/p3/tareap3ant/principal)
> ==4608==    by 0x1094D8: main (in /mnt/c/p3/tareap3ant/principal)
> ==4608==  If you believe this happened as a result of a stack
> ==4608==  overflow in your program's main thread (unlikely but
> ==4608==  possible), you can try to increase the size of the
> ==4608==  main thread stack using the --main-stacksize= flag.
> ==4608==  The main thread stack size used in this run was 8388608.
> 1>timeout: the monitored command dumped core
> Segmentation fault (core dumped)
