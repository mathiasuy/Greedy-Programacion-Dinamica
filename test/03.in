max_cantidad_intervalos 
30
1-15-1
2-8-1
15-30-1
27-35-1
20-33-1
9-15-1
10-22-1
25-40-1
45-63-1
50-57-1
4-35-1
10-42-1
30-58-1
61-64-1
63-75-1
59-63-1
63-70-1
43-75-1
60-78-1
67-70-1
8-15-1
30-33-1
39-55-1
25-43-1
38-40-1
48-77-1
65-77-1
74-78-1
34-42-1
50-68-1
Fin
