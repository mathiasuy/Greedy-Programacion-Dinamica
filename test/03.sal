==4385== error calling PR_SET_PTRACER, vgdb might block
1>