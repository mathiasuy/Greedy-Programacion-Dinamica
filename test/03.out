1>Cantidad: 9
2>Fin.
